src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,88 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd(
+    movie: Movie
+  ): void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+  const [description, setDescription] = useState('');
```

Comment:
consider to make one state for all inputs
```suggestion
  
   const [newMovie, setNewMovie] = useState({
     title: '',
     description: '',
     imgUrl: '',
     imdbUrl: '',
     imdbId: '',
   });
   
   const onChange = (event: ChangeEvent<HTMLInputElement>) => {
     const { name, value } = event.target;

      setNewMovie(prevMovie => ({ ...prevMovie, [name]: value }));
   };
```